Sources/MintKit/Mint.swift

```diff
@@ -268,6 +268,24 @@ public class Mint {
             }
         }
 
+        // copy manpages
+        do {
+            let manpages = Path("\(packageCheckoutPath)/doc/man/").glob("*")
+            let allFiles = try manpages
+                .flatMap { try $0.recursiveChildren() }
+                .map { $0.lastComponent }
+                .joined(separator: ", ")
+
+            standardOut <<< "🌱  Copying manpages for \(package.name): \(allFiles) ..."
```

Comment:
If there are no man files can we not print this message

Reply:
Done in 75619de